docs/content/docs/try-flink/local_installation.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: 'Local Installation'
+title: 'First steps'
 weight: 1
 type: docs
 aliases:
-  - /try-flink/local_installation.html
+  - /try-flink/first_steps.html
```

Comment:
Please revert. The aliases are redirects from past locations of this page. They should never be changed, just updated if we want to move or rename this page in the future.

Reply:
Yes, I had wanted to rename the whole file but then just did half the steps